src/audio/src/src.c

```diff
@@ -484,6 +481,21 @@ static void src_copy_sxx(struct comp_dev *dev,
 }
 
 #if CONFIG_IPC_MAJOR_4
+static int src_get_attribute(struct comp_dev *dev, uint32_t type, void *value)
```

Comment:
needs to have a size since it copies data to void *

Reply:
@lgirdwood I can add the change to add size in a folloow up PR. This will unblock ipc4 topologies tests that have a gain module